fpmsyncd/routesync.cpp

```diff
@@ -747,6 +754,79 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
 }
 
 /* 
+ * Handle label route
+ * @arg nlmsg_type      Netlink message type
+ * @arg obj             Netlink object
+ */
+void RouteSync::onLabelRouteMsg(int nlmsg_type, struct nl_object *obj)
+{
+    struct rtnl_route *route_obj = (struct rtnl_route *)obj;
+    struct nl_addr *daddr;
+    char destaddr[MAX_ADDR_SIZE + 1] = {0};
+
```

Comment:
check route table index, if 'non zero' throw error and return.

Reply:
Done